pkgs/development/tools/build-managers/build2/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv, lib, fetchurl, sqlite }:
+
+stdenv.mkDerivation rec {
+  pname = "build2";
+  version = "0.13.0";
+
+  src = fetchurl {
+    url = "https://download.build2.org/${version}/build2-toolchain-${version}.tar.xz";
+    sha256 = "01hmr5y8aa28qchwy9ci8x5q746flwxmlxarmy4w9zay9nmvryms";
+  };
+
+  dontConfigure = true;
+  dontInstall = true;
+
+  buildPhase = ''
+    runHook preBuild
+    ./build.sh --local --trust yes --install-dir "$out" "$CXX"
+    runHook postBuild
+  '';
+
+  meta = with lib; {
+    homepage = "https://www.build2.org/";
+    description = ''
+      An open source, cross-platform build
+      toolchain that aims to approximate Rust Cargo's convenience
+    '';
```

```suggestion
    description = "build2 build system";
```
Might be a little too long for a one-liner. I like to use the package subtitle from cppget (https://cppget.org/build2).